### PR TITLE
[WIP] Remove xml fields

### PR DESCRIPTION
**Pull request by andygnewman**
cc: @i-like-robots 

Remove dependency on XML fields from ES.

Should wait for bodyHTML for V1 articles to be fully deployed, before merging in.

### CODE REVIEW

File: server/controllers/article.js [hunk 1]
```diff
@@ -104,12 +104,9 @@ module.exports = function articleV3Controller (req, res, next, content) {
 		res.set('X-Robots-Tag', 'noindex');
 	}
 
-	// If no bodyHTML, revert to using bodyXML
-	const contentToTransform = content.bodyHTML || content.bodyXML;
-
 	// Apply content and article specific transforms to bodyHTML
-	if (contentToTransform) {
-		Object.assign(content, transformArticleBody(contentToTransform, res.locals.flags, {
+	if (content.bodyHTML) {
```
Comment (i-like-robots) [inline server/controllers/article.js]: Remove the conditional entirely?

File: test/server/controllers/article-helpers/myft-metadata.test.js [hunk 2]
```diff
@@ -7,7 +7,6 @@ const fixtureEsFound = require('../../../fixtures/v3-elastic-article-found').doc
 
 const subject = proxyquire('../../../../server/controllers/article', {
 	'./article-helpers/suggested': () => Promise.resolve(),
-	'../transforms/article-xslt': (article) => Promise.resolve(article.bodyXML),
```
Comment (i-like-robots) [inline test/server/controllers/article-helpers/myft-metadata.test.js]: oh wow

Comment (andygnewman) [inline test/server/controllers/article-helpers/myft-metadata.test.js]: Was that a general 'oh wow' or one that should result in a further code change?

Comment (i-like-robots) [inline test/server/controllers/article-helpers/myft-metadata.test.js]: Just amazed to see it here!

**Pull request merged**